bindings/wasm/src/wasm_did.rs

```diff
@@ -11,49 +11,49 @@ use crate::utils::err;
 /// @typicalname did
 #[wasm_bindgen(inspectable)]
 #[derive(Clone, Debug, PartialEq)]
-pub struct DID(pub(crate) IotaDID);
+pub struct WasmDID(pub(crate) IotaDID);
```

Comment:
you can use [`#[wasm_bindgen(js_name = DID)]`](https://rustwasm.github.io/docs/wasm-bindgen/reference/attributes/on-rust-exports/js_name.html) here to rename the JavaScript type

https://github.com/iotaledger/identity.rs/discussions/218#discussioncomment-665049

Reply:
Done (I think, plz double check 🙂)